libpod/runtime_ctr.go

```diff
@@ -221,17 +221,18 @@ func (r *Runtime) newContainer(ctx context.Context, rSpec *spec.Spec, options ..
 
 // RemoveContainer removes the given container
 // If force is specified, the container will be stopped first
+// If removeVolume is specified, the volume contents will be removed also
```

Comment:
"If removeVolume is specified, named volumes used by the container will be removed also if and only if the container is the sole user"

Reply:
Fixed